### DISCUSSION TITLE
重定向早年域名和网址

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1628] [功能建议 / Ideas]
如早年的视频页 URL 格式可能有好几种：
- `https://www.bilibili.tv/video/<aid>`
- `https://www.bilibili.com/video/<aid>.html`

还有目前还能用但还是建议重定向下的个人空间页：
`https://space.bilibili.com/<uid>#!/index`

以前没怎么留意过，可能还有别的格式。

**Comment by Geeyun-JY3** [comment]
旧版的视频页格式还有：`https://www.bilibili.tv/video/<aid>/index.html`
- 发现于：https://weibo.com/2125605343/zfVafhMeS

个人空间页也有目前不能用的格式：`http://space.bilibili.tv/<uid>`
- 发现于：https://site.douban.com/gz180/

目前破站把 bilibili.tv 用作国际站，可能还得考虑一下如何不干扰到访问国际站。
上去看了一下，国际站上的视频播放页和个人空间页的 URL 格式还是不一样的。

**Reply by AzusaHana** [reply]
`https://www.bilibili.tv/video/<id>` 

如果<id>是纯数字，且国际站有该视频，会跳转 `https://www.bilibili.tv/en/video/<id>` ，自动加上显示语言，

像 av号 这种还是会跳转 国际站 但显示404
